server/text_generation_server/models/custom_modeling/flash_santacoder_modeling.py

```diff
@@ -74,8 +74,17 @@ def _load_multi_mqa_gptq(
         qzeros = torch.cat([q_tensor, kv_tensor], dim=1)
 
         g_idx = weights.get_tensor(f"{prefix}.c_attn.g_idx")
-        bits = weights.get_tensor("gptq_bits").item()
-        groupsize = weights.get_tensor("gptq_groupsize").item()
+        try:
+            bits = weights.get_tensor("gptq_bits").item()
+            groupsize = weights.get_tensor("gptq_groupsize").item()
+        except SafetensorError as e:
+            try:
+                import os
+
+                bits = int(os.getenv("GTPQ_BITS"))
```

Comment:
I think there's a typo here "GTPQ" instead of "GPTQ"

Reply:
Thanks !

Reply:
@Narsil, I've just created a pull request (https://github.com/huggingface/text-generation-inference/pull/590) to fix this and also to repeat this logic in the function below where it's also required.